tests/Content/TranslationTest.php

```diff
@@ -0,0 +1,361 @@
+
+		$this->createContentMultiLanguage();
+
+		$this->assertTrue($translationEN->version()->exists());
+		$this->assertTrue($translationDE->version()->exists());
+	}
+
+	/**
+	 * @covers ::exists
+	 */
+	public function testExistsSingleLanguage()
+	{
+		$this->setUpSingleLanguage();
+
+		$translation = new Translation(
+			model: $this->model,
+			version: $this->model->version(),
+			language: Language::single()
+		);
+
+		$this->assertFalse($translation->version()->exists());
+
+		$this->createContentSingleLanguage();
+
+		$this->assertTrue($translation->version()->exists());
+	}
+
+	/**
+	 * @covers ::isDefault
+	 */
+	public function testIsDefault()
+	{
+		$this->setUpMultiLanguage();
+
+		$en = new Translation(
+			model: $this->model,
+			version: $this->model->version(),
+			language: Language::ensure('en')
+		);
+
+		$de = new Translation(
+			model: $this->model,
+			version: $this->model->version(),
+			language: Language::ensure('de')
+		);
+
+		$this->assertTrue($en->language()->isDefault());
+		$this->assertFalse($de->language()->isDefault());
+	}
+
+	/**
+	 * @covers ::language
+	 */
+	public function testLanguage()
+	{
+		$this->setUpMultiLanguage();
+
+		$translationEN = new Translation(
+			model: $this->model,
+			version: $this->model->version(),
+			language: $languageEN = Language::ensure('en')
+		);
+
+		$translationDE = new Translation(
+			model: $this->model,
+			version: $this->model->version(),
+			language: $languageDE = Language::ensure('de')
+		);
+
+		$this->assertSame($languageEN, $translationEN->language());
+		$this->assertSame($languageDE, $translationDE->language());
+	}
+
+	/**
+	 * @covers ::model
+	 */
+	public function testModel()
+	{
+		$this->setUpMultiLanguage();
+
+		$translation = new Translation(
+			model: $this->model,
+			version: $this->model->version(),
+			language: Language::ensure('en')
+		);
+
+		$this->assertSame($this->model, $translation->model());
+	}
+
+	/**
+	 * @covers ::slug
+	 */
+	public function testSlugExists()
+	{
+		$this->setUpMultiLanguage();
+
+		$translationEN = new Translation(
+			model: $this->model,
+			version: $this->model->version(),
+			language: Language::ensure('en')
+		);
+
+		$translationDE = new Translation(
+			model: $this->model,
+			version: $this->model->version(),
+			language: Language::ensure('de')
+		);
+
+		Data::write($this->contentFile('en'), [
+			'title' => 'Test',
+			'slug'  => 'english-slug'
+		]);
+
+		Data::write($this->contentFile('de'), [
+			'title' => 'Test',
+			'slug'  => 'german-slug'
+		]);
+
+		$this->assertSame('english-slug', $translationEN->slug());
+		$this->assertSame('german-slug', $translationDE->slug());
+	}
+
+	/**
+	 * @covers ::slug
+	 */
+	public function testSlugNotExists()
+	{
+		$this->setUpMultiLanguage();
+
+		$translation = new Translation(
+			model: $this->model,
+			version: $this->model->version(),
+			language: Language::ensure('en')
+		);
+
+		Data::write($this->contentFile('en'), [
```

Comment:
This should then also test the secondary language for consistency:

```suggestion
			language: Language::ensure('de')
		);

		Data::write($this->contentFile('de'), [
```